### DISCUSSION TITLE
en/setting-domain-for-dynamic-ip-address-openwrt-ddns/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#124] [General]
# en/setting-domain-for-dynamic-ip-address-openwrt-ddns/

Free world, free thoughts, free words.

https://dariusz.wieckiewicz.org/en/setting-domain-for-dynamic-ip-address-openwrt-ddns/

**Comment by kostas-77** [comment]
Very helpful. Thank you.
